src/oemof/solph/components/_generic_storage.py

```diff
@@ -56,19 +58,19 @@ class GenericStorage(Node):
         Absolute nominal capacity of the storage, fixed value or
         object describing parameter of investment optimisations.
     invest_relation_input_capacity : numeric (iterable or scalar) or None, :math:`r_{cap,in}`
-        Ratio between the investment variable of the input Flow and the
+        Ratio between the investment variable of the input flow and the
```

Comment:
I think these had capital letters to refer to the class `Flow`.

Reply:
I also have no clear preference. I just wanted to make sure that it is not changed before we have a common understanding of the previous way. In my option, 2 and 3 are cleaner than 1.